zerver/webhooks/rundeck/view.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Dict, Sequence
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import webhook_view
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+RUNDECK_MESSAGE_TEMPLATE = "**{name}** - {status} - [E{id}]({link})"
```

Comment:
I think we should make better use of Markdown formatting and topics here. Would it make sense to use the name as the topic, and a bit more of a sentence in the message format?